### DISCUSSION TITLE
I have a question about the CIS code repo "add-on".  Would we be able to add that to our existing RefArch codebase?  Do you have any more documentation or lessons-learned re: bolting this on that goes into more detail than https://gruntwork.io/achieve-compliance ?

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#329] [Help] [answered]
I have a question about the CIS code repo "add-on".  Would we be able to add that to our existing RefArch codebase?  Do you have any more documentation or lessons-learned re: bolting this on that goes into more detail than https://gruntwork.io/achieve-compliance ?

**Comment by autero1** [comment, marked as answer]
This was answered in https://github.com/gruntwork-io/knowledge-base/discussions/316
